src/renderer/stores/settings.ts

```diff
@@ -195,14 +219,17 @@ onReady(() => {
       SettingsStoreInstance.reducer.setSelfContact(selfContact)
     }
     window.__updateAccountListSidebar?.()
-  })
-  BackendRemote.on('ConfigSynced', (_accountId, { key }) => {
+  }
+  // SelfavatarChanged event is deprecated and does not seem to work (neither SelfavatarChanged nor iver the new ConfigSynced)
```

Comment:
@iequidoo I couldn't get self avatar syncing to work at all in my testing.

If it works and also triggers the `ConfigSynced` event correctly, then we cn remove the listener for the deprecated SelfavatarChanged event?